app/lib/screens/login_screen.dart

```diff
@@ -15,6 +15,8 @@ import 'package:provider/provider.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 
 import 'wait_screen.dart';
+import 'package:manvsim/utils/platform_checker_web.dart'
+    if (dart.library.io) 'package:manvsim/utils/platform_checker.dart';
```

Comment:
Seems like the `dart.library.io` is a library not available on web.  
Similiarly `dart.library.html` is only available on web.  
I think it would be more understandable swapped:  
```dart
import 'package:manvsim/utils/platform_checker.dart'
    if (dart.library.html) 'package:manvsim/utils/platform_checker_web.dart';
```

Reply:
Done